api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,57 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      get:
+        security: []
+        tags:
+          - concepts
+
+        description: >
+          Searches for concepts in concept table by name, and optionally filter on
+          domain ID, vocabulary ID, or standard concept status. Uses the CDR version affiliated
+          with the workspace specified in the path.
+
+        operationId: "searchConcepts"
+        parameters:
+          - in: query
+            name: query
+            type: string
+            required: true
+            description: >
+              A query string that can be used to match a subset of the name (case-insensitively),
+              the entire code value (case-insensitively), or the concept ID.
+          - in: query
+            name: standard_concept
+            type: boolean
+            required: false
```

Comment:
I'm pretty sure false is the default, would remove these

Reply:
Done.